tslint.json

```diff
@@ -69,7 +69,7 @@
         "component-selector": [true, "element", "hc", "kebab-case"],
         "use-input-property-decorator": true,
         "use-output-property-decorator": true,
-        "use-host-property-decorator": true,
+        "use-host-property-decorator": false,
```

Comment:
While I understand the reasoning for preferring `@HostBinding` in a regular front-facing project, in our case where we're using unencapsulated styles and `:host` isn't available to us, I believe the convenience of the `host` property in the Component annotation outweighs the downsides for adding static host classes.